crates/rust-client/src/rpc/mod.rs

```diff
@@ -127,8 +127,8 @@ pub trait NodeRpcClient: Send + Sync {
     /// Fetches note-related data for a list of [`NoteId`] using the `/GetNotesById` rpc endpoint.
     ///
     /// For any `NoteType::Private` note, the return data is only the
```

Comment:
We should also wrap this one
```suggestion
    /// For any `[miden_objects::note::NoteType::Private]` note, the return data is only the
```

Reply:
Great catches!